Packs/Cryptosim/IncidentFields/incidentfield-Cryptosim_Category_ID.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@haldunerbay Why not using the existing "Category" field instead?

Reply:
Category ID is specific key for cryptosim alerts, so category mean more general name.

Reply:
If you think it's critical, you can add it, but please add a prefix to the cliname, id and name of the field and also associate it only to the relevant incident type.

Reply:
Added.